.github/ISSUE_TEMPLATE/4-Issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Issue (for development only)
+about: As a developer you want to describe an issue
+---
+
+## Description
+
+<!--- Please describe the issue in few words -->
```

Comment:
<!--- Please describe the issue in few words -->
=> (not sure)
<!--- Please describe the issue in a few words -->